wdl/transforms/biscayne/src/test/scala/wdl/transforms/biscayne/linking/expression/values/BiscayneValueEvaluatorSpec.scala

```diff
@@ -46,7 +46,7 @@ class BiscayneValueEvaluatorSpec extends AnyFlatSpec with CromwellTimeoutSpec wi
   it should "evaluate an 'as_pairs' expression correctly" in {
     // Repeat this a few times ensure we aren't occasionally reordering by accident:
     10 times {
-      val str = """ as_pairs( { 1: "one", 2: "two", 3: three } ) """
+      val str = """ as_pairs( { 1: "one", 2: "two", 3: "three" } ) """
```

Comment:
As mentioned on the `cascades` side, I _think_ this was deliberately a variable lookup rather than a plain string

Reply:
Undone, good catch. Consistency is not always the only goal, as it turns out 🙂